Cargo.toml

```diff
@@ -22,7 +22,8 @@ tempfile = "3.0"
 slog = "2.1.1"
 slog-stdlog = "3.0.2"
 libloading = "0.5.0"
-wayland-client = { version = "0.23.4", features = ["egl"], optional = true }
+wayland-client = { version = "0.23.4", optional = true }
```

Comment:
This is still the old wayland-client...

Reply:
Ah right.

Actually, since this change, `smithay` no longer depends on `wayland-client`, only `wayland-egl`. So this line can be removed altogether :sweat_smile: